outline/intro.md

```diff
@@ -195,7 +195,7 @@ Introduction to Programming with Clojure
 
 **R**ead, **E**val, **P**rint, **L**oop
 
-![Instarepl](img/instarepl.png)
+![Nightcode's repl](img/repl.png)
```

Comment:
@yokolet, I'm happy to make this change in a subsequent pull request. Shall we merge this pull request as is?